Create SJET-AOCODARCF722V2.config

ChenYang78256 opened the pull request:
1.Serial port 1 Use  receiver
2.Serial port 2 Use a GPS
3.The features of OSD、GPS are enabled by default
4. The receiver selects CRSF protocol by default
5. The default motor protocol is Dshot600 by default
6.Bluetooth are enabled by default
7.Use the User1 and User2 mode
8.Add User1 mode. User1 mode Control Bluetooth switch
9.Add User2 mode. User2 mode Controls the switchover between cam1 and cam2
9.MSP for Bluetooth enabled serial port 5 by default

ChenYang78256 committed:
Create SJET-AOCODARCF722APPV2.config

YangChen1234 commented:
The function of LED strip and GPS cannot be turned on using your own locally compiled 4.4.3 firmware, but it can be turned on by tapping the online download of AOCODARCF7DUAL 4.4.3 firmware and tapping the F722APP's profile.

lida2003 commented:
> The function of LED strip and GPS cannot be turned on using your own locally compiled 4.4.3 firmware, 

I don't know what kind of binary you are using. I think it's good to use releases form bf team. Below binary is what I have got from their [4.4.3 release note](https://github.com/betaflight/betaflight/releases/tag/4.4.3). I also noticed that they have remove the binary from their website. 

![图片](https://github.com/Aocoda-RC/unified-targets/assets/1296171/c06095a6-d86f-4e42-a7f1-ed8becf530da)



> but it can be turned on by tapping the online download of AOCODARCF7DUAL 4.4.3 firmware and tapping the F722APP's profile.

The best way is to use a stable version for you verified if it's OK or NOT. 

PS: If you have concerns, please check with bf team, as the above binary is NOT a custom build version.

lida2003 commented on configs/default/SJET-AOCODARCF722APPV2.config:
```diff
@@ -0,0 +1,156 @@
+set dshot_burst = ON
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 118
+set ibata_offset = -620
```
Please remove custom current settings, it's NOT an AIO board.

lida2003 commented on configs/default/SJET-AOCODARCF722APPV2.config:
```diff
@@ -0,0 +1,156 @@
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name AOCODARCF722APPV2
```
Make sure the product is Aocoda-RC F722 APP V2.

lida2003 reviewed (changes requested):
Please do the final test and check those comments, thanks.

ChenYang78256 committed:
Update SJET-AOCODARCF722APPV2.config

1.Change the name of the board to AOCODARCF722V2
2.Remove current and  battery settings

ChenYang78256 committed:
Rename SJET-AOCODARCF722APPV2.config to SJET-AOCODARCF722V2.config

YangChen1234 commented:
A BUG that could not open LED strips and GPS was submitted
https://github.com/betaflight/betaflight/issues/13645

ChenYang78256 committed:
Update SJET-AOCODARCF722V2.config

ChenYang78256 committed:
Update SJET-AOCODARCF722V2.config

YangChen1234 commented:
self test passed

YangChen1234 commented:
self test passed

ChenYang78256 committed:
Update SJET-AOCODARCF722V2.config

ChenYang78256 committed:
Update SJET-AOCODARCF722V2.config

ChenYang78256 committed:
Update SJET-AOCODARCF722V2.config

YangChen1234 commented:
self test passed

YangChen1234 commented:
The config file has passed final functional and flight tests.
1. The CRSF and SBUS functions of the receiver are passed
2. The ESC callback passed
3.CAM,VTX and OSD passed
4. Motor output is normal
5.GYRO,BARO, and MAG passed
6.ADC,GPS,LEDS,BZ passed
7.  Bluetooth function passed
8. Flight test is stable and smooth
